src/webapp/src/Navbar.tsx

```diff
@@ -20,12 +21,20 @@ const Header = styled.div`
   margin: 13.5px auto;
 `;
 
-export const Navbar = () => (
-  <HeaderStyles>
-    <Header>
-      <Button size='medium' onClick={() => {}}>
-        Login
-      </Button>
-    </Header>
-  </HeaderStyles>
-);
+export const Navbar = () => {
+  const history = useHistory();
+  return (
+    <HeaderStyles>
```

Comment:
Looks like there is a border missing on the bottom of the navbar. Mind taking a moment in this PR to add it?
![image](https://user-images.githubusercontent.com/4181674/109998607-4f16bd00-7d09-11eb-8a79-eac2ebe8e74b.png)

Reply:
The border appear and disappear in the home route depending on the position of the scroll bar. In the login routes is where is always there. I have to find a way of doing that nicely.

Reply:
We can start by always having it there and find a way to remove it only in certain cases